### DISCUSSION TITLE
No visual indicator if a cell was executed

### DISCUSSION THREAD

**Discussion by Yelinz** [marimo-team/marimo#677] [Ideas]
AFAIK the only way to tell if a cell was executed is if the execution timer is > 0.
Which is not very intuitive or easy

**Comment by mscolnick** [comment]
marimo notebooks aim to always be up-to-date with the latest state. Rather than having some indicator (besides the timer) to show when a cell is executed, we actually show when it is stale - the cell is yellow-ish and faded. 

So instead, you should assume a cell is always executed, unless shown as stale. Does that help?
